src/Entities/PaymentRetryInstructions/PaymentInstruction.php

```diff
@@ -28,7 +28,7 @@
     const PARTIAL = 'partial';
     const DISCOUNT = 'discount';
 
-    const AVAILABLE_METHODS = [
+    private static $availableMethods = [
```

Comment:
Why this?

Reply:
@slavcodev It is incompatible with PHP 5.5:
http://php.net/manual/en/migration56.new-features.php

Reply:
Oh, we supports 5.5 here. I think we can check API log and check the minimal PHP version used by our clients.

Reply:
Sure. But it should be separated PR and (probably) new SDK version. For now i think it is correct to support minimum declared PHP version compatibility

Reply:
Sure.